scripts/install_dependencies.py

```diff
@@ -71,7 +71,11 @@ def git_clone(repo_url, branch, dst, recursive=False):
               False,
               "bb-runtimes",
               False,
-              [sys.executable, ROOT_DIR + "/bb-runtimes/install.py", "--arch=arm-eabi"]),
+              [sys.executable, ROOT_DIR + "/bb-runtimes/install.py",
+               "--arch=arm-eabi",
+               "--prefix=" +
```

Comment:
(minor) why not use the .format() rather than string concatenation, for homogeneity with other places in the script?

Reply:
I find it more readable in this case.